website_media_size/static/src/js/website_media_size.js

```diff
@@ -0,0 +1,33 @@
+/* Copyright 2018 Onestein
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
+
+odoo.define('website_media_size', function(require) {
+    "use strict";
+
+    var MediaDialog = require('web_editor.widget').MediaDialog;
+    var weContext = require("web_editor.context");
+    var Widget = require('web.Widget');
+
+    MediaDialog.include({
+        xmlDependencies: MediaDialog.prototype.xmlDependencies.concat(
+            ['/website_media_size/static/src/xml/website_media_size.xml']
+        ),
+        start: function() {
+            var res = this._super.apply(this, arguments);
+            this.search();
+            return res;
+        }
+    });
+
+    // ImageDialog of the web_editor.widget module is not available
```

Comment:
@tarteo https://github.com/odoo/odoo/pull/23180 got merged. Do you still need this?

Reply:
No, had no time to change it yet.